### PR TITLE
enh(lang): add translations

### PR DESCRIPTION
**Pull request by victorvassilev**
## Description

Add extra translations to exception errors in api v2.

**Fixes** # MON-5270

## Type of change

- [ ] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software
- [x] Updating documentation (missing information, typo...)

## Target serie

- [ ] 2.8.x
- [ ] 18.10.x
- [ ] 19.04.x
- [ ] 19.10.x
- [x] 20.04.x (master)

<h2> How this pull request can be tested ? </h2>

Test translations.

## Checklist

- [x] I followed the **coding style guidelines** provided by Centreon
- [x] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [x] I have commented my code, especially **hard-to-understand areas** of the PR.
- [x] I have made corresponding changes to the **documentation**.
- [x] I have **rebased** my development branch on the base branch (master, maintenance).

### CODE REVIEW

File: lang/fr_FR.UTF-8/LC_MESSAGES/messages.po [hunk 1]
```diff
@@ -14842,3 +14842,314 @@ msgstr "Eléments verrouillés"
...
+msgid "Service %1 on host %2 not found"
+msgstr "Service %1 de l'hôte %2 non trouvé"
+
+#:Centreon/Application/Controller/GorgoneController.php:142
+msgid "Unrecognized Command"
+msgstr "Commande non reconnue"
+
+#:Centreon/Application/Controller/GorgoneController.php:157
+msgid "Commands containing a request body are not allowed at the moment"
+msgstr "Les commandes contenant un \"request body\" ne sont pas autorisés pour le moment"
```
Comment (bdauria) [inline lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:14893]:
```suggestion
msgstr "Les commandes contenant un \"request body\" ne sont pas autorisées pour le moment"
```

File: lang/fr_FR.UTF-8/LC_MESSAGES/messages.po [hunk 2]
```diff
@@ -14842,3 +14842,314 @@ msgstr "Eléments verrouillés"
...
+
+#:Centreon/Domain/Engine/EngineService.php:105
+#:Centreon/Domain/Engine/EngineService.php:149
+#:Centreon/Domain/Engine/EngineService.php:146
+msgid "The host of service is not defined"
+msgstr "L'hôte du service n'est pas défini"
+
+#:Centreon/Domain/Engine/EngineService.php:172
+msgid "Host of downtime not found"
+msgstr "L'hôte associé à la plage de maintenance non trouvé"
```
Comment (bdauria) [inline lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:14976]:
```suggestion
msgstr "L'hôte associé à la plage de maintenance n'a pas été trouvé"
```

File: lang/fr_FR.UTF-8/LC_MESSAGES/messages.po [hunk 3]
```diff
@@ -14842,3 +14842,314 @@ msgstr "Eléments verrouillés"
...
+msgid "Host name of service %1 can not be empty"
+msgstr "Le nom d'hôte du service %1 ne peut pas être vide"
+
+#:Centreon/Domain/Engine/EngineService.php:241
+msgid "The description of service %1 can not be empty"
+msgstr "La description du service %1 ne peut pas être vide"
+
+#:Centreon/Domain/Engine/EngineService.php:286
+msgid "Host and service id can not be null at the same time"
+msgstr "L'ID de l'hôte et du service ne peuvent être null en même temps"
```
Comment (bdauria) [inline lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:14992]:
```suggestion
msgstr "L'ID de l'hôte et du service ne peuvent pas être null en même temps"
```

File: lang/fr_FR.UTF-8/LC_MESSAGES/messages.po [hunk 4]
```diff
@@ -14842,3 +14842,314 @@ msgstr "Eléments verrouillés"
...
+
+#:Centreon/Domain/Engine/EngineService.php:289
+msgid "Downtime internal id can not be null"
+msgstr "L'ID de l'hôte et du service ne peuvent être null en même temps"
```
Comment (bdauria) [inline lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:14996]:
```suggestion
msgstr "L'ID de l'hôte et du service ne peuvent pas être null en même temps"
```

File: lang/fr_FR.UTF-8/LC_MESSAGES/messages.po [hunk 5]
```diff
@@ -14842,3 +14842,314 @@ msgstr "Eléments verrouillés"
...
+
+#:Centreon/Domain/Engine/EngineService.php:355
+msgid "Host name cannot be empty"
+msgstr "Le nom d'hôte ne peut être vide"
```
Comment (bdauria) [inline lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:15000]:
```suggestion
msgstr "Le nom d'hôte ne peut pas être vide"
```

File: lang/fr_FR.UTF-8/LC_MESSAGES/messages.po [hunk 6]
```diff
@@ -14842,3 +14842,314 @@ msgstr "Eléments verrouillés"
...
+
+#:Centreon/Domain/Engine/EngineService.php:359
+msgid "Service description cannot be empty"
+msgstr "La description du service ne peut être vide"
```
Comment (bdauria) [inline lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:15004]:
```suggestion
msgstr "La description du service ne peut pas être vide"
```

File: lang/fr_FR.UTF-8/LC_MESSAGES/messages.po [hunk 7]
```diff
@@ -14842,3 +14842,314 @@ msgstr "Eléments verrouillés"
...
+
+#:Centreon/Domain/Exception/ExpiredTokenException.php:29
+msgid "Token expired."
```
Comment (bdauria) [inline lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:15007]: Are . allowed in msgids?

File: lang/fr_FR.UTF-8/LC_MESSAGES/messages.po [hunk 8]
```diff
@@ -14842,3 +14842,314 @@ msgstr "Eléments verrouillés"
...
+msgstr "Token expiré."
+
+#:Centreon/Domain/Exception/ContactDisabledException.php:29
+msgid "Contact disabled."
+msgstr "Contact désactivé."
+
+#:Centreon/Domain/Gorgone/ActionLog.php:77
+msgid "Token can not empty, null or not defined"
+msgstr "Le token ne peut êter null, vide ou non défini"
```
Comment (bdauria) [inline lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:15016]:
```suggestion
msgstr "Le token ne peut être ni null, ni vide ou non défini"
```

File: lang/fr_FR.UTF-8/LC_MESSAGES/messages.po [hunk 9]
```diff
@@ -14842,3 +14842,314 @@ msgstr "Eléments verrouillés"
...
+
+#:Centreon/Domain/Gorgone/GorgoneService.php:69
+msgid "Error when connecting to the Gorgone server"
+msgstr "Erreur sde connexion avec le server Gorgone"
```
Comment (bdauria) [inline lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:15020]:
```suggestion
msgstr "Erreur de connexion avec le serveur Gorgone"
```

File: lang/fr_FR.UTF-8/LC_MESSAGES/messages.po [hunk 10]
```diff
@@ -14842,3 +14842,314 @@ msgstr "Eléments verrouillés"
...
+msgid "Error while searching for resources"
+msgstr "Erreur lors de la recherche de la ressource"
+
+#:Centreon/Domain/MonitoringServer/MonitoringServerService.php:57
+msgid "Error when searching for monitoring servers"
+msgstr "Erreur lors de la recherche des évènements"
+
+#:Centreon/Domain/MonitoringServer/MonitoringServerService.php:69
+msgid "Error when searching for a resource of monitoring server"
+msgstr "Erreur lors de la recherche de la ressource attaché au serveur de supervision"
```
Comment (bdauria) [inline lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:15048]:
```suggestion
msgstr "Erreur lors de la recherche de la ressource attachée au serveur de supervision"
```

File: lang/fr_FR.UTF-8/LC_MESSAGES/messages.po [hunk 11]
```diff
@@ -14842,3 +14842,314 @@ msgstr "Eléments verrouillés"
...
+msgstr "Le modèle %1 n'a pas été trouvé"
+
+#:Centreon/Infrastructure/Monitoring/MonitoringRepositoryRDB.php:189
+#:Centreon/Infrastructure/Monitoring/MonitoringRepositoryRDB.php:535
+#:Centreon/Infrastructure/Monitoring/MonitoringRepositoryRDB.php:590
+#:Centreon/Infrastructure/Monitoring/MonitoringRepositoryRDB.php:702
+#:Centreon/Infrastructure/Monitoring/MonitoringRepositoryRDB.php:802
+#:Centreon/Infrastructure/Monitoring/MonitoringRepositoryRDB.php:1010
+msgid "Bad SQL request"
+msgstr "Le modèle %1 n'a pas été trouvé"
```
Comment (bdauria) [inline lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:15114]:
```suggestion
msgstr "Mauvaise requête SQL"
```

File: lang/fr_FR.UTF-8/LC_MESSAGES/messages.po (hunk 7 above)
Comment (victorvassilev) [reply lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:15007]: they actually are allowed, but it is not good practice to use them. We have several of this kind in the translation file and this is something that can be improved for sure.

File: lang/fr_FR.UTF-8/LC_MESSAGES/messages.po [hunk 12]
```diff
@@ -14840,6 +14840,316 @@ msgstr "Eléments verrouillés"
...
+msgstr "La date de fin doit être supérieure à la date de début."
+
+#:Centreon/Application/Controller/AcknowledgementController.php:573
+msgid "Incorrect resource type for disacknowledgement"
+msgstr "Type de ressource incorrect pour le désacquittement"
+
+#:Centreon/Application/Controller/AcknowledgementController.php:650
+#:Centreon/Application/Controller/DowntimeController.php:644
+msgid "Incorrect resource type for acknowledgement"
+msgstr "Type de ressource incorrect pour le l'acquittement"
```
Comment (bdauria) [inline lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:14873]:
```suggestion
msgstr "Type de ressource incorrect pour l'acquittement"
```

Comment (kduret) [inline lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:14873]:
```suggestion
msgstr "Type de ressource incorrect pour l'acquittement"
```

File: lang/fr_FR.UTF-8/LC_MESSAGES/messages.po [hunk 13]
```diff
@@ -14840,6 +14840,316 @@ msgstr "Eléments verrouillés"
...
+msgid "Acknowledgement already cancelled for this host"
+msgstr "Acquittement déjà annulé pour cet hôte"
+
+#:Centreon/Domain/Acknowledgement/AcknowledgementService.php:275
+msgid "No acknowledgement found for this service"
+msgstr "Aucun acquittement trouvé pour ce service"
+
+#:Centreon/Domain/Acknowledgement/AcknowledgementService.php:278
+msgid "Acknowledgement already cancelled for this service"
+msgstr "Acquittement déjà annulé pour cet service"
```
Comment (cgagnaire) [inline lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:14923]:
```suggestion
msgstr "Acquittement déjà annulé pour ce service"
```

File: lang/fr_FR.UTF-8/LC_MESSAGES/messages.po [hunk 14]
```diff
@@ -14840,6 +14840,316 @@ msgstr "Eléments verrouillés"
...
+msgid "Incorrect Resource type: %1"
+msgstr "Type de ressource incorrect: %1"
+
+#:Centreon/Domain/ConfigurationLoader/YamlConfigurationLoader.php:102
+msgid "Loop detected in file %1"
+msgstr "Boucle détectée dans le fichier %1"
+
+#:Centreon/Domain/ConfigurationLoader/YamlConfigurationLoader.php:137
+msgid "The configuration file %1 does not exists"
+msgstr "Le fichier de configuration %1 n'éxiste pas"
```
Comment (cgagnaire) [inline lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:14935]:
```suggestion
msgstr "Le fichier de configuration %1 n'existe pas"
```

File: lang/fr_FR.UTF-8/LC_MESSAGES/messages.po [hunk 15]
```diff
@@ -14840,6 +14840,316 @@ msgstr "Eléments verrouillés"
...
+#:Centreon/Domain/Downtime/DowntimeService.php:267
+msgid "Incorrect Resource Type"
+msgstr "Type de ressource incorrect"
+
+#:Centreon/Domain/Engine/EngineService.php:76
+#:Centreon/Domain/Engine/EngineService.php:102
+#:Centreon/Domain/Engine/EngineService.php:169
+#:Centreon/Domain/Engine/EngineService.php:220
+msgid "The contact alias is empty"
+msgstr "Le \"contact alias\" est vide"
```
Comment (cgagnaire) [inline lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:14954]:
```suggestion
msgstr "L'alias du contact est vide"
```

File: lang/fr_FR.UTF-8/LC_MESSAGES/messages.po [hunk 16]
```diff
@@ -14840,6 +14840,316 @@ msgstr "Eléments verrouillés"
...
+msgid "The description of service %1 can not be empty"
+msgstr "La description du service %1 ne peut pas être vide"
+
+#:Centreon/Domain/Engine/EngineService.php:286
+msgid "Host and service id can not be null at the same time"
+msgstr "L'ID de l'hôte et du service ne peuvent pas être null en même temps"
+
+#:Centreon/Domain/Engine/EngineService.php:289
+msgid "Downtime internal id can not be null"
+msgstr "L'ID de l'hôte et du service ne peuvent pas être null en même temps"
```
Comment (cgagnaire) [inline lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:14993]:
```suggestion
msgstr "L'ID interne du temps d'arrêt ne peut pas être null"
```

File: lang/fr_FR.UTF-8/LC_MESSAGES/messages.po [hunk 17]
```diff
@@ -14840,6 +14840,316 @@ msgstr "Eléments verrouillés"
...
+#:Centreon/Domain/Exception/ExpiredTokenException.php:29
+msgid "Token expired."
+msgstr "Token expiré."
+
+#:Centreon/Domain/Exception/ContactDisabledException.php:29
+msgid "Contact disabled."
+msgstr "Contact désactivé."
+
+#:Centreon/Domain/Gorgone/ActionLog.php:77
+msgid "Token can not empty, null or not defined"
```
Comment (cgagnaire) [inline lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:15012]:
```suggestion
msgid "Token can not be empty, null or not defined"
```

Comment (cgagnaire) [reply lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:15012]: Might broker translate.

File: lang/fr_FR.UTF-8/LC_MESSAGES/messages.po [hunk 18]
```diff
@@ -14840,6 +14840,316 @@ msgstr "Eléments verrouillés"
...
+msgstr "Le token ne peut être ni null, ni vide ou non défini"
+
+#:Centreon/Domain/Gorgone/GorgoneService.php:69
+msgid "Error when connecting to the Gorgone server"
+msgstr "Erreur de connexion avec le serveur Gorgone"
+
+#:Centreon/Domain/HostConfiguration/HostConfigurationService.php:52
+msgid "Host name already exists"
+msgstr "Le nom d'hôte éxiste déjà"
```
Comment (quanghungb) [inline lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:15021]: existe

File: lang/fr_FR.UTF-8/LC_MESSAGES/messages.po [hunk 19]
```diff
@@ -14840,6 +14840,316 @@ msgstr "Eléments verrouillés"
...
+msgid "Invalid credentials"
+msgstr "Identifiants invalides"
+
+#:Centreon/Application/Controller/DowntimeController.php:203
+#:Centreon/Application/Controller/DowntimeController.php:333
+msgid "Service %1 on host %2 not found"
+msgstr "Service %1 de l'hôte %2 non trouvé"
+
+#:Centreon/Application/Controller/GorgoneController.php:142
+msgid "Unrecognized Command"
```
Comment (cgagnaire) [inline lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:14885]:
```suggestion
msgid "Unrecognized command"
```

Comment (cgagnaire) [reply lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:14885]: Might break translate.

File: lang/fr_FR.UTF-8/LC_MESSAGES/messages.po (hunk 17 above)
Comment (victorvassilev) [reply lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:15012]: yeah, the key is already in place, let's not change it

File: lang/fr_FR.UTF-8/LC_MESSAGES/messages.po [hunk 20]
```diff
@@ -14840,6 +14840,316 @@ msgstr "Eléments verrouillés"
...
+msgid "Error when connecting to the Gorgone server"
+msgstr "Erreur de connexion avec le serveur Gorgone"
+
+#:Centreon/Domain/HostConfiguration/HostConfigurationService.php:52
+msgid "Host name already exists"
+msgstr "Le nom d'hôte éxiste déjà"
+
+#:Centreon/Domain/HostConfiguration/HostConfigurationService.php:61
+msgid "Error while creation of host"
+msgstr "Le nom d'hôte éxiste déjà"
```
Comment (quanghungb) [inline lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:15025]: existe

Comment (cgagnaire) [inline lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:15025]:
```suggestion
msgstr "Erreur lors de la creation de l'hôte"
```

File: lang/fr_FR.UTF-8/LC_MESSAGES/messages.po [hunk 21]
```diff
@@ -14840,6 +14840,316 @@ msgstr "Eléments verrouillés"
...
+msgid "Token can not empty, null or not defined"
+msgstr "Le token ne peut être ni null, ni vide ou non défini"
+
+#:Centreon/Domain/Gorgone/GorgoneService.php:69
+msgid "Error when connecting to the Gorgone server"
+msgstr "Erreur de connexion avec le serveur Gorgone"
+
+#:Centreon/Domain/HostConfiguration/HostConfigurationService.php:52
+msgid "Host name already exists"
+msgstr "Le nom d'hôte éxiste déjà"
```
Comment (cgagnaire) [inline lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:15021]:
```suggestion
msgstr "Le nom d'hôte existe déjà"
```

File: lang/fr_FR.UTF-8/LC_MESSAGES/messages.po [hunk 22]
```diff
@@ -14840,6 +14840,316 @@ msgstr "Eléments verrouillés"
...
+msgstr "Contact non trouvé"
+
+#:Centreon/Domain/Security/AuthenticationService.php:113
+#:Centreon/Infrastructure/Gorgone/CommandRepositoryAPI.php:98
+msgid "Token not found"
+msgstr "Token non trouvé"
+
+#:Centreon/Infrastructure/Engine/EngineRepositoryFile.php:87
+msgid "Centcore directory %1 does not exist"
+msgstr "Token non trouvé"
```
Comment (quanghungb) [inline lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:15078]: The message is completly different. Is it normal ?

File: lang/fr_FR.UTF-8/LC_MESSAGES/messages.po [hunk 23]
```diff
@@ -14840,6 +14840,316 @@ msgstr "Eléments verrouillés"
...
+msgid "Error while searching for the number of host"
+msgstr "Erreur lors de la recherche du nombre d'hôte"
+
+#:Centreon/Domain/Monitoring/ResourceService.php:102
+msgid "Error while searching for resources"
+msgstr "Erreur lors de la recherche de la ressource"
+
+#:Centreon/Domain/MonitoringServer/MonitoringServerService.php:57
+msgid "Error when searching for monitoring servers"
+msgstr "Erreur lors de la recherche des évènements"
```
Comment (cgagnaire) [inline lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:15041]:
```suggestion
msgstr "Erreur lors de la recherche de serveurs de supervision"
```

File: lang/fr_FR.UTF-8/LC_MESSAGES/messages.po [hunk 24]
```diff
@@ -14840,6 +14840,316 @@ msgstr "Eléments verrouillés"
...
+msgid "Error while searching for the host"
+msgstr "Erreur lors de la recherche de l'hôte"
+
+#:Centreon/Domain/HostConfiguration/HostConfigurationService.php:85
+msgid "Error while searching for the number of host"
+msgstr "Erreur lors de la recherche du nombre d'hôte"
+
+#:Centreon/Domain/Monitoring/ResourceService.php:102
+msgid "Error while searching for resources"
+msgstr "Erreur lors de la recherche de la ressource"
```
Comment (cgagnaire) [inline lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:15037]:
```suggestion
msgstr "Erreur lors de la recherche de ressources"
```

File: lang/fr_FR.UTF-8/LC_MESSAGES/messages.po [hunk 25]
```diff
@@ -14840,6 +14840,316 @@ msgstr "Eléments verrouillés"
...
+msgid "Error while searching for the web version of the Centreon platform"
+msgstr "Erreur lors de la récupération de la version de la plate-forme Centreon"
+
+#:Centreon/Domain/Platform/PlatformService.php:70
+msgid "Error while searching for the modules version of the Centreon platform"
+msgstr "Erreur lors de la recherche de la version des extensions"
+
+#:Centreon/Domain/Platform/PlatformService.php:82
+msgid "Error while searching for the widgets version of the Centreon platform"
+msgstr "Erreur lors de la recherche de la version du widget"
```
Comment (cgagnaire) [inline lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:15061]:
```suggestion
msgstr "Erreur lors de la recherche de la version des widgets"
```

File: lang/fr_FR.UTF-8/LC_MESSAGES/messages.po [hunk 26]
```diff
@@ -14840,6 +14840,316 @@ msgstr "Eléments verrouillés"
...
+msgstr "Contact non trouvé"
+
+#:Centreon/Domain/Security/AuthenticationService.php:113
+#:Centreon/Infrastructure/Gorgone/CommandRepositoryAPI.php:98
+msgid "Token not found"
+msgstr "Token non trouvé"
+
+#:Centreon/Infrastructure/Engine/EngineRepositoryFile.php:87
+msgid "Centcore directory %1 does not exist"
+msgstr "Token non trouvé"
```
Comment (cgagnaire) [inline lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:15078]:
```suggestion
msgstr "Le répertoire Centcore %1 n'existe pas"
```

File: lang/fr_FR.UTF-8/LC_MESSAGES/messages.po (hunk 22 above)
Comment (quanghungb) [reply lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:15078]: Le dossier %1 de Centcore n'existe pas

File: lang/fr_FR.UTF-8/LC_MESSAGES/messages.po [hunk 27]
```diff
@@ -14840,6 +14840,316 @@ msgstr "Eléments verrouillés"
...
+msgid "Error during deleting expired session"
+msgstr "Erreur lors de l'expiration de la session"
+
+#:Centreon/Infrastructure/Security/AuthenticationRepositoryRDB.php:214
+msgid "Error when recovering the number of tokens"
+msgstr "Erreur lors de la récupération du nombre de token"
+
+#:Centreon/Infrastructure/Security/AuthenticationRepositoryRDB.php:232
+msgid "Error when deleting tokens for one contact"
+msgstr "Erreur lors de la récupération du nombre de token"
```
Comment (cgagnaire) [inline lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:15152]:
```suggestion
msgstr "Erreur lors de la suppression des tokens pour un contact"
```

File: lang/fr_FR.UTF-8/LC_MESSAGES/messages.po [hunk 28]
```diff
@@ -14840,6 +14840,316 @@ msgstr "Eléments verrouillés"
...
+msgid "Service %1 on host %2 not found"
+msgstr "Service %1 de l'hôte %2 non trouvé"
+
+#:Centreon/Application/Controller/GorgoneController.php:142
+msgid "Unrecognized Command"
+msgstr "Commande non reconnue"
+
+#:Centreon/Application/Controller/GorgoneController.php:157
+msgid "Commands containing a request body are not allowed at the moment"
+msgstr "Les commandes contenant un \"request body\" ne sont pas autorisées pour le moment"
```
Comment (cgagnaire) [inline lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:14890]:
```suggestion
msgstr "Les commandes contenant un corp ne sont pas autorisées pour le moment"
```

File: lang/fr_FR.UTF-8/LC_MESSAGES/messages.po [hunk 29]
```diff
@@ -14840,6 +14840,316 @@ msgstr "Eléments verrouillés"
...
+msgid "The description of service %1 can not be empty"
+msgstr "La description du service %1 ne peut pas être vide"
+
+#:Centreon/Domain/Engine/EngineService.php:286
+msgid "Host and service id can not be null at the same time"
+msgstr "L'ID de l'hôte et du service ne peuvent pas être null en même temps"
+
+#:Centreon/Domain/Engine/EngineService.php:289
+msgid "Downtime internal id can not be null"
+msgstr "L'ID interne du temps d'arrêt ne peut pas être null"
```
Comment (cgagnaire) [inline lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:14993]:
```suggestion
msgstr "L'ID interne de la plage de maintenance ne peut pas être null"
```

Comment (cgagnaire) [reply lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:14993]: My bad, change the french for consistency.

**Pull request merged**